packages/api/utils/validation.test.ts

```diff
@@ -186,6 +189,54 @@ describe("isEnvironment()", () => {
     expect(isEnvironment(mockEnvironmentsMissingFields)).toEqual(false);
   });
 });
+describe("isOperator()", () => {
+  const missingOperatorFields = [
+    {
+      // display_name: "name",
+      email: "yoda@iam.mil",
+      access: "administrator",
+    },
+    {
+      display_name: "name",
+      // email: "yoda@iam.mil",
+      access: "administrator",
+    },
+    {
+      display_name: "name",
+      email: "yoda@iam.mil",
+      // access: "administrator",
+    },
+  ];
+
+  it.each([true, 1, undefined, null])("should reject non-object", (item) => {
+    expect(isOperator(item)).toEqual(false);
+  });
+  it.each(mockApplicationJabbasPalaceExpansionApp.operators)("should accept an Operator object", (item) => {
+    expect(isOperator(item)).toEqual(true);
+  });
+  it.each(missingOperatorFields)("should reject Operator with missing field", (item) => {
+    expect(isOperator(item)).toEqual(false);
+  });
+});
+describe("isMilEmail()", () => {
+  const goodMilEmails = [
+    "good-email@testing.mil",
+    "good.email_1234567890@testing.mil",
+    "good_email_1234567890@testing_1234567-890.mil",
+  ];
+  const badMilEmails = [
+    "bad.email_1234567890@testing.au",
+    "bad-email!@#$%^&*()+=@testing.co",
```

Comment:
If you were to take out the parentheses and change the TLD to `.mil` this actually could be valid